tex/artikel/Geld.tex

```diff
@@ -179,21 +182,21 @@ \subsection{Befreiung vom Rundfunkbeitrag}
 Ausgenommen sind nur Gartenlauben und Wohnwagen.
 Auch in Zimmern in Jugendherbergen besteht keine Beitragspflicht.
 
-Haushalt heißt nun, dass nicht mehr jeder Rundfunk-Teilnehmer und Beitragszahler ist, sondern nur noch der Inhaber der Wohnung.
-Und auch hier meint das Gesetz nicht den Besitzer, sondern jede volljährige Person, die in dieser Wohnung gemeldet oder deren Mieter er ist.
+Haushalt heißt nun, dass nicht mehr jede Person Beitragszahler*in ist, sondern nur noch die*der Inhaber*in der Wohnung.
+Und auch hier meint das Gesetz jede volljährige Person, die in dieser Wohnung gemeldet ist oder dort zur Miete wohnt.
```

Comment:
Das hier sollte mit der Änderung im vorherigen Satz bereits klar sein, und der Satz verwirrt ehrlich gesagt nach der Änderung in Kombination mit dem vorherigen: Nicht jede Person, sondern nur die inhabende Person, aber doch irgendwie jede volljährige Person, die in der Wohnung gemeldet ist... Sollte in jedem Fall jetzt auch ohne den Satz klar sein.
```suggestion
```